crates/duty-tracker/src/contract_state_machine.rs

```diff
@@ -750,13 +750,15 @@ impl ContractState {
     /// Maps the claim_txid to the operator's p2p key.
     pub fn claim_to_operator(&self, claim_txid: &Txid) -> Option<P2POperatorPubKey> {
         let claim_txids = match self {
-            ContractState::Requested { claim_txids, .. } => claim_txids,
-            ContractState::Deposited { claim_txids, .. } => claim_txids,
-            ContractState::Assigned { claim_txids, .. } => claim_txids,
-            ContractState::StakeTxReady { claim_txids, .. } => claim_txids,
-            ContractState::Fulfilled { claim_txids, .. } => claim_txids,
-            ContractState::Claimed { claim_txids, .. } => claim_txids,
-            ContractState::Challenged { claim_txids, .. } => claim_txids,
+            ContractState::Requested { claim_txids, .. }
+            | ContractState::Deposited { claim_txids, .. }
+            | ContractState::Assigned { claim_txids, .. }
+            | ContractState::StakeTxReady { claim_txids, .. }
+            | ContractState::Fulfilled { claim_txids, .. }
+            | ContractState::Claimed { claim_txids, .. }
+            | ContractState::Challenged { claim_txids, .. }
+            | ContractState::PreAssertConfirmed { claim_txids, .. }
+            | ContractState::AssertDataConfirmed { claim_txids, .. } => claim_txids,
```

Comment:
I don't mind this too terribly much but I do think it's worse than what it used to be on the grounds that it's harder to edit now and it reads a little less fluidly.

Reply:
I find easier to use the `|` guide in the left and know that I only have one output in the right `claim_txids` instead of what we had before that I had to visually pattern match all the things in the right to make sure that they were all `claim_txids`.